metadata.php

```diff
@@ -172,6 +172,16 @@
             'block' => 'email_plain_order_owner_orderemail',
             'file' => 'views/blocks/email_plain_order_owner_orderemail.tpl'
         ),
+        array(
+            'template' => 'payment_main.tpl',
+            'block' => 'admin_payment_main_editor',
+            'file' => 'views/admin/blocks/admin_payment_main_editor.tpl'
+        ),
+        array(
+            'template' => 'module_config.tpl',
```

Comment:
Indentation is not consistent here – either indent the block above to be on the same column or this one to not be.